app/i18n/i18n.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+  detection: {
+    order: ["htmlTag"],
+    lookupCookie: "i18next",
+  },
+  // This is the list of languages your application supports
+  supportedLngs: ["en", "fr"],
```

Comment:
Here again we are repeating this languages. This will be tricky to maintain. We need a central way to manage what languages we have. Maybe this should be stored in the shelf.config file.